configs/redis.json

```diff
@@ -0,0 +1,64 @@
+{
+  "index_name": "redis",
+  "start_urls": [
+    {
+      "url": "https://docsearch-redis.netlify.com/documentation"
+    },
+    {
+      "url": "https://docsearch-redis.netlify.com/commands",
+      "selectors_key": "commands",
+      "page_rank": 5
+    },
+    {
+      "url": "https://docsearch-redis.netlify.com/topics",
+      "selectors_key": "topics",
+      "page_rank": 1
+    }
+  ],
+  "stop_urls": [],
+  "strip_chars": "*",
+  "selectors": {
+    "default": {
```

Comment:
There is no real `default` page, everything is either a command or a topic, so I added the two custom selector entries and set the default to a set of empty selectors.

I tried removing the `default` key but the crawling failed.

What do you suggest putting instead of empty selectors? something like `#lkdjqkksqdkjhqsjkdkjsq`?

Reply:
I would recommend to set selectors in order to have something that will evolve properly.

Reply:
I don't see how I could do that. There is no "default" page, they are either topics or commands, and each require their own specific selectors.

What selector would you suggest I add?

Reply:
Not yet but it can happrn later

Reply:
Using the same selector

Reply:
> Using the same selector

I don't get that. The same selector as what? Do you mean copy-pasting the same selectors as the `topics` or `commands` into the `default`?

Reply:
Yes

Reply:
But which one then? `topics` or `commands`? And I don't see why it's better than empty selectors. We have no idea what the new pages (that don't exist yet) will look like, so no idea if those selectors will match anything

Reply:
The more generic one

Reply:
That's the thing :) They're both specific, none of them is generic.

And I tried to remove the `default` key, but they DocSearch was giving me errors, so the key needs to be present. It's just that I don't know what to put in the selectors. I put `#will-never-match` (or something similar) to document that those selectors are garbage and not meant to be used. See it as a way to document the hack

Reply:
Fine with it. You can merge and deploy it when you need